### PR TITLE
Feature play development card

### PR DESCRIPTION
**Pull request by alemazzo**
- General Refactor
- Implemented playability of development card

### CODE REVIEW

File: src/main/scala/scatan/views/game/components/DevelopmentCardPopups.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      display <-- this.toBeShown.signal.map(if _ then "block" else "none"),
+      position.fixed,
+      top("50%"),
+      left("50%"),
+      transform := "translate(-50%, -50%)",
+      padding := "20px",
+      backgroundColor := "white",
+      boxShadow := "0 2px 10px rgba(0, 0, 0, 0.1)",
+      borderRadius := "5px",
+      zIndex := 1000,
```
Comment (luigi-borriello00) [inline src/main/scala/scatan/views/game/components/DevelopmentCardPopups.scala:25]: Please move this style attributes in `style.css` file :)

File: src/main/scala/scatan/views/game/components/DevelopmentCardPopups.scala [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      div(
+        title,
+        textAlign.center,
+        fontSize := "20px",
+        margin := "10px"
```
Comment (luigi-borriello00) [inline src/main/scala/scatan/views/game/components/DevelopmentCardPopups.scala:30]: Also this 💯

**Review by luigi-borriello00 — CHANGES_REQUESTED** [review]
All perfect, except the style attributes in Laminar api :)

**Review by manuandru — APPROVED** [review]
I think this is the last "big" feature  🙌

**Pull request closed without merging**